orca-queue-tck/src/main/kotlin/com/netflix/spinnaker/orca/q/ExecutionLatch.kt

```diff
@@ -43,7 +43,7 @@ class ExecutionLatch(private val predicate: Predicate<ExecutionComplete>)
     }
   }
 
-  fun await() = latch.await(1, TimeUnit.SECONDS)
+  fun await() = latch.await(2, TimeUnit.SECONDS)
```

Comment:
Thank you!  I was just trying to figure out why that test was intermittently failing.

Reply:
no problem. I still don't know why travis has troubles with this test - 1s should be plenty... but we just decided to bump it 🤞